support-frontend/app/admin/settings/Settings.scala

```diff
@@ -39,7 +39,7 @@ object Settings {
   def fromS3[T: Decoder](source: SettingsSource.S3)(implicit s3: AwsS3Client): Either[Throwable, T] =
     for {
       buf <-
-        s3.fetchAsString(new AmazonS3URI("s3://" + source.bucket + "/" + source.key))
+        s3.fetchAsString(S3Location(source.bucket, "/" + source.key))
```

Comment:
the new `S3Uri` class requires an instance of the S3 client to parse an s3 uri - not sure if it makes an api call?  Also the bucket and key are optional in it.  So I switched to a normal case class where possible.

https://docs.aws.amazon.com/AmazonS3/latest/userguide/example_s3_Scenario_URIParsing_section.html